### PR TITLE
fix dropdown

### PR DESCRIPTION
**Pull request by AnnaGoodman1**

### CODE REVIEW

**Comment by AnnaGoodman1** [comment]
also now updated outputs_regrional_sha to fix this issue https://github.com/npct/pct-shiny/issues/738

**Comment by Robinlovelace** [comment]
Thanks @AnnaGoodman1 I'll take a look today and try to get a review by 5pm.

**Comment by Robinlovelace** [comment]
Tested and looks fine to me: lines under a scenario below

image

**Comment by Robinlovelace** [comment]
Order Lines disappears gracefully when Census 2011 is selected:

image

**Comment by Robinlovelace** [comment]
@nikolai-b I could merge this but leave to you to take a final look before hitting the Merge pull request button. Thanks.

File: regions_www/m/ui.R [hunk 1]
```diff
@@ -136,7 +136,7 @@ shinyUI(
                 )
               ),
               conditionalPanel(
-                condition = "['route_network', 'olc'].indexOf(input.line_type) === -1",
+                condition = "['route_network'].indexOf(input.line_type) === -1 & ['olc'].indexOf(input.scenario) === -1 ",
```
[the remark this answers is not in the document]
Comment (AnnaGoodman1) [reply regions_www/m/ui.R]: ah thanks, hadn't understood about  (/had never bothered to look up) the 2 types of & before

**Comment by AnnaGoodman1** [comment]
@nikolai-b ,could you now put this change on live server when you have a minute? Thanks, Anna

**Pull request merged**